### ISSUE TITLE
Crashes every time

### ISSUE THREAD

**Issue by michaloslav** [StarsoftAnalysis/pug-pdf#3]
Here's what I get whenever I run the example command (the one listed under Usage):
internal/streams/legacy.js:59
      throw er; // Unhandled stream error in pipe.
      ^

Error: Command failed: F:\Programming\Node.js\pribehy\node_modules\phantomjs-prebuilt\lib\phantom\bin\phantomjs.exe F:\Programming\Node.js\pribehy\node_modules\pug-pdf\phantom-script.js C:\Users\MICHAL~1\AppData\Local\Temp\tmp-12216Ov3qxDBcoP9x.html C:\Users\MICHAL~1\AppData\Local\Temp\tmp-122166LjBVy51151Q.pdf F:\Programming\Node.js\pribehy\node_modules\pug-pdf/../pdf.css A4 portrait 1cm

    at ChildProcess.exithandler (child_process.js:275:12)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at maybeClose (internal/child_process.js:925:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)

**Comment by juan-sb** [comment]
Have you ever solved this somehow?

**Comment by michaloslav** [comment]
> Have you ever solved this somehow?

No, I ended up using [email-templates](https://www.npmjs.com/package/email-templates)
